scripts/start-shared-test-validator.sh

```diff
@@ -22,7 +24,7 @@ TEST_VALIDATOR_LEDGER="$( cd "$(dirname "${BASH_SOURCE[0]}")/.." ; pwd -P )/test
       wait
     else
       echo "Sharing lock on already running test validator (PID $(pidof $TEST_VALIDATOR))"
-      sleep infinity
+      while true; do sleep 86400; done
```

Comment:
I'm concerned. The idea here is that `sleep infinity` emits a file descriptor that forms part of the lock. If `sleep 86400` finishes, does it temporarily release the lock before the loop restarts?

Yes, I know 86400 seconds is a whole day :)

Reply:
Ah, got you. How about we just do like `sleep 86400000` here? I don't know what the macOS actual max is but it doesn't complain about that